docs/source/changelog.rst

```diff
@@ -20,6 +20,7 @@ This release contains a large number of bug fixes and small improvements, and
 sets the stage for the major cycling improvement (:issue:`750`) coming in the
 next release in a few days.
 
+* Added line wrapping for title. by `@Sylviabohnenstengel`_ in :pr:`935`
```

Comment:
We no longer need to manually write the changelog, as I now generate it from the pull request title. It doesn't hurt though so you can just leave it here.